### ISSUE TITLE
Quick Forms: Only assets with the status 'Active' appear.

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#585]
Please can we update the quick forms so that:

Only _Active_ Land assets appear in the Locations box on all quick forms.
Only _Active_ Plant and Structural assets appear and can be selected. 

Specifically, if an assets is Harvested, Discontinued or Archived then it no longer displays in the quick form list

**Comment by paul121** [comment]
Hey @aislinnpearson, I'm looking at the logic in the query and it seems to already be designed like this, only display `active` assets. I'm not sure how an archived asset would be displayed as an option. Do you have an example/test to demonstrate this?

**Comment by aislinnpearson** [comment]
Hey @paul121,

that's great to know (but terrible that I can't remember what we've done so far). 

I think the issue might stem from my end then. I am hesitant to archive any of our crop assets until they have been checked by the farm team so I have left them active until the point where we can mark them as 'completed' or 'harvested' (#586). In short, I haven't tested the logic and didn't realise it was already there (althoguh admittedly I should have - but I think I wanted to make sure that we were also capturing the use case in #586).

I have just realised that experiment boundaries are land assets, so also need the status 'Completed' (not just 'Discontinued') so I updated issue #586.

One to chat about this evening I think.
